langchain/base_language.py

```diff
@@ -29,7 +28,7 @@ def _get_token_ids_default_method(text: str) -> List[int]:
     return tokenizer.encode(text)
 
 
-class BaseLanguageModel(BaseModel, ABC):
+class BaseLanguageModel(Serializable, ABC):
```

Comment:
Serializable base class should replace BaseModel (and always be the first in the list, to ensure `__init__` gets called